### ISSUE TITLE
Exception: System.ArgumentOutOfRangeException

### ISSUE THREAD

**Issue by aelizj** [PowerShell/PSReadLine#1672]
```
Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.      
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo 
key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)    
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics) ``
```

**Comment by aelizj** [comment]
It happened again: 

```
Last 193 Keys:
 r u b y Space " c : \ U s e r s \ a 
n n e 1 \ L a u n c h S c h o o l \ R B 1 0 1 \ 2 n d _ P a s s _ R u b y 
_ S m a l l _ P r o b l e m s \ e a s y _ 2 \ r o o m _ s i z e . r b " Enter
 i r b Enter
 c l s Enter
 r u b y Space " c : \ U s e r s \ a 
n n e 1 \ L a u n c h S c h o o l \ R B 1 0 1 \ 2 n d _ P a s s _ R u b y 
_ S m a l l _ P r o b l e m s \ e a s y _ 2 \ t i p _ c a l c u l a t o r 
. r b " Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.      
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo 
key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)    
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)

```

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.2 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
